src/routes/(infoPages)/land/fire/safety/fire-danger-map/+page.server.ts

```diff
@@ -1,6 +1,22 @@
+import gql from "graphql-tag";
+import { print as printQuery } from "graphql";
 import parishTopoJSON from "./parishes.json";
 
-export const load = async ({ locals: { getKVClient } }) => {
+import type { FireWeatherMapQuery } from "./$queries.generated";
+
+const query = gql`
+  query FireWeatherMap {
+    fireDangerMapCollection(limit: 1) {
+      items {
+        description {
+          json
+        }
+      }
+    }
+  }
+`;
```

Comment:
This could have an adverse side-effect if there's more than one entry erroneously created; I'd recommend doing something like we did on `/licensing-permits` where we look up the  pageMetadata ID based on the route (and then query for the one entry instead of a collection): https://github.com/la-ldaf/ldaf-site/blob/9aa03aa9241b5f32bad23bbb8245454944269393/src/routes/licensing-permits/%2Bpage.server.ts#L95